src/rculfhash.c

```diff
@@ -568,6 +568,43 @@ unsigned int cds_lfht_fls_ulong(unsigned long x)
 #endif
 }
 
+static
+void *cds_lfht_malloc(void *state __attribute__((unused)), size_t size)
+{
+	return malloc(size);
+}
+
+static
+void *cds_lfht_calloc(void *state __attribute__((unused)), size_t nmemb, size_t size)
+{
+	return calloc(nmemb, size);
+}
+
+static
+void *cds_lfht_realloc(void *state __attribute__((unused)), void *ptr, size_t size)
+{
+	return realloc(ptr, size);
+}
+
+static
+void *cds_lfht_aligned_alloc(void *state __attribute__((unused)), size_t alignment, size_t size)
+{
+	return aligned_alloc(alignment, size);
```

Comment:
aligned_alloc() requires C11. C99 only has posix_memalign() afaik.

Reply:
Do you see merit in changing this to `posix_memalign()` or should I completely remove it?

Reply:
I think that exposing aligned_alloc in the public API makes sense, but internally in our default implementation of cds_lfht_aligned_alloc we should use posix_memaglin() rather than aligned_alloc() because urcu builds with c99 as well.

Reply:
I replaced the default implementation to use posix_memalign() instead.